Sources/Contour/Caching/MemoryCacheProvider.cs

```diff
@@ -1,153 +0,0 @@
-﻿namespace Contour.Caching
-{
-    using System;
-    using System.Runtime.Caching;
-
-    using Contour.Helpers;
-    using Contour.Helpers.CodeContracts;
-
-    /// <summary>
-    /// The memory cache provider.
-    /// </summary>
-    public class MemoryCacheProvider : ICacheProvider
```

Comment:
Breaking backward compatilibity. You should avoid it.

Reply:
Reverted deletion and marked as obsolete.